### PR TITLE
aura EIP updates

### LINKED ISSUE #1: EIP-AuRa: Checklist before getting started
- [x] create EIP for AuRa PoA from temlate (see EIP-X.md)
- [x] review the Clique PoA EIP-225, see https://github.com/ethereum/EIPs/issues/225
- [x] review the AuRa spec on Parity wiki https://wiki.parity.io/Aura.html
- [ ] request EIP number from upstream

### LINKED ISSUE #4: Aura open questions
> - What is the block format -- is it compatible?
> - Where is the block seal? How is it calculated? (e.g, `hash(rlp(blockheader_without_seal_field)))` ?

---

> - Does the consensus engine have to perform a call to a contract at each block, to obtain validator list?
> - No current support for this in Geth atm, and there is no notion of `SYSTEM_ADDRESS` in the codebase.

---

> - Geth supports code in the genesis, but not 'constructor'. So instead of hacking that in, might be easier to just calculate the runtime bytecode and the storage slots, and stash them directly into the genesis.
> - I think the hardest part may be to implement the interfacing needed for the consensus engine to reach into the EVM to evaluate a contract call. Implementing 'Immutable list'-configuration is a lot easier, it's pretty close to Clique.

### PR DESCRIPTION
**Pull request by 5chdn**
answering questions and fixed spelling and grammar

closes #1 

closes #4

### CODE REVIEW

File: EIPS/eip-aura.md [hunk 1]
```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
 ---
 
 ## Simple Summary
-Aura is a basic consensus algorithm where only a defined set of users is able to _mine_ blocks.
+Aura is a basic consensus algorithm where only a defined set of users can _mine_ blocks.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: you can remove can.

File: EIPS/eip-aura.md [hunk 2]
```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
...
+Aura is the consensus engine pioneered by [Parity Technologies](https://parity.io) in 2016 and used to power the [public Kovan testnet](https://medium.com/@Digix/announcing-kovan-a-stable-ethereum-public-testnet-10ac7cb6c85f) which was established after the [Ropsten spam-attacks](https://ethereum.stackexchange.com/questions/12477/ropsten-testnet-is-under-kind-of-attack-what-can-we-do) in 2017. Besides, public value-bearing networks, such as the [POA Network](https://poa.network/), are based on the Aura protocol today.
 
 This document is _informational_ to summarize all available technical specification of the consensus engine to allow other client developer teams an overview, a place for feedback, and potentially the foundation to implement this in other code bases.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: This document is _informational_  and summarizes the available technical specification of the consensus engine. This provides other client developers teams an overview...

File: EIPS/eip-aura.md [hunk 3]
```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
...
 
 ## Motivation
-Networks not bearing any value can not be sufficiently secured with proof-of-work or proof-of-stake. This is especially true for test networks that are supposed to be a reliable platform for developers to deploy and test their applications. Therefore, a proof-of-authority scheme is proposed which provides sufficient resistance against network attacks in testnets without attached value.
+Networks not bearing any value cannot be sufficiently secured with proof-of-work or proof-of-stake. This circumstance is uniquely right for test networks that are supposed to be a reliable platform for developers to deploy and test their applications. Therefore, a proof-of-authority scheme is proposed which provides sufficient resistance against network attacks in testnets without any value attached.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: Networks bearing no value cannot...

File: EIPS/eip-aura.md [hunk 4]
```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
...
 
 ### Authority Round Description
 Time is divided into discrete steps of duration `t`, determined by:
 
 ![Formula 00](../assets/eip-aura/00-unix-time.png)
 
-At each step `s`, a _primary_ will be assigned. Only the primary at a step may issue a block. It is misbehavior to produce more than one block per step or to produce a block out of turn.
-
-The primary for a step `s` is the node with index:
+At each step `s`, a _primary_ is assigned. Only the primary at a step may issue a block. It is misbehavior to produce more than one block per step or to produce a block out of turn. The primary for a step `s` is the validator node with index:
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: consider making the two additional _primary_'s italicized.

Comment (ChainSafe) [inline EIPS/eip-aura.md]: 3rd sentence - Producing more than one block per step or to produce a block out of turn is bad behavior.

File: EIPS/eip-aura.md [hunk 5]
```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
...
 
 ![Formula 01](../assets/eip-aura/01-s-bmod-n.png)
 
-The protocol contains a chain scoring rule `SCORE(C)` for a given chain `C`.
-
-On each step, each honest node will propagate the chain with the highest score it knows about to all other nodes. Honest primaries will only issue blocks on top of the best chain they are aware of during their turn.
+The protocol contains a chain scoring rule `SCORE(C)` for a given chain `C`. On each step, each honest node propagates the chain with the highest score it knows about to all other nodes. Honest primaries only issue blocks on top of the best chain they are aware of during their turn.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: At each step, each honest node propagates the chain with the highest score to all other nodes.

File: EIPS/eip-aura.md [hunk 6]
```diff
@@ -54,16 +87,16 @@ If there is a valid chain `C` ending with `C[K..]`, where:
...
 ![Formula 03](../assets/eip-aura/03-sig-set-ck.png)
 
-then `C[K]` and all of its ancestors are finalized.
-
-This definition of finality stems from a simple majority vote. In this setting:
+then `C[K]` and all of its ancestors are finalized. This definition of finality stems from a simple majority vote. In this setting:
 
 ![Formula 04](../assets/eip-aura/04-2f-leq-n.png)
 
 so the faulty nodes cannot finalize a block all on their own.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: the faulty node cannot finalize a block alone.

File: EIPS/eip-aura.md [hunk 7]
```diff
@@ -54,16 +87,16 @@ If there is a valid chain `C` ending with `C[K..]`, where:
...
 
 ### Configuration
-This consensus requires a _Validator Set_ to be specified, which determines the list of `n` blockchain addresses at each height `h` which participate in the consensus. Validators are a group of accounts which are allowed to participate in the consensus, they validate the transactions and blocks to later sign messages about them. The validator set can be specified in a number of different ways.
+This consensus requires a _Validator Set_ to be specified, which determines the list of `n` blockchain addresses at each height `h` which participate in the consensus. Validators are a group of accounts which are allowed to participate in the consensus; they validate the transactions and blocks to sign messages about them later.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: they validate the transactions and blocks, signing messages about them later.

File: EIPS/eip-aura.md [hunk 8]
```diff
@@ -54,16 +87,16 @@ If there is a valid chain `C` ending with `C[K..]`, where:
...
+
+The validator set can be specified in different ways. To implement a basic Authority Round engine, an _immutable list_ is sufficient. However, optionally, it can be dynamically provided by smart contracts containing the validator sets. Multisets as implemented in Parity Ethereum even allow combining hard-forking from immutable to dynamic validator sets.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: Implementing a basic...

File: EIPS/eip-aura.md [hunk 9]
```diff
@@ -78,9 +111,9 @@ A simple list of addresses specified at genesis, i.e.:
 ```
 
 #### Contracts
-The list can be also a part of the blockchain state by being stored in an Ethereum contract.
+_(Optional:)_ The list can be also a part of the blockchain state by being stored in an Ethereum contract.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: The list can also be a part of the...

File: EIPS/eip-aura.md [hunk 10]
```diff
@@ -78,9 +111,9 @@ A simple list of addresses specified at genesis, i.e.:
...
 
-It is best to include the contract in the genesis placing the bytecode as a "constructor" in the "accounts" field. If the constructor takes arguments they must be encoded and appended to the contract bytecode (using e.g. [ethabi](https://github.com/paritytech/ethabi)). Also if the contract initializes any address with `msg.sender` (for example as a contract owner) you must take into account that when defining the contract in genesis, `msg.sender` will be set to the system address (`SYSTEM_ADDRESS`: `2^160 - 2`).
+It is best to include the contract in the genesis placing the bytecode as a "constructor" in the "accounts" field. If the constructor takes arguments, they must be encoded and appended to the contract bytecode (using, e.g., [`ethabi`](https://github.com/paritytech/ethabi)). Also if the contract initializes any address with `msg.sender` (for example as a contract owner) you must take into account that when defining the contract in genesis, `msg.sender` is set to the system address (`SYSTEM_ADDRESS`: `2^160 - 2`).
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: ...genesis, placing...

File: EIPS/eip-aura.md [hunk 11]
```diff
@@ -117,27 +150,24 @@ contract ValidatorSet {
...
 
-The function `getValidators` should always return the active set or the initial set if the contract hasn't been activated yet.
-Switching the set should be done by issuing a `InitiateChange` event with the parent block hash and new set, storing the pending set, and then waiting for call to `finalizeChange` (by the `SYSTEM_ADDRESS`: `2^160 - 2`) before setting the active set to the pending set. This mechanism is used to ensure that the previous validator set "signs off" on the changes before activation, leading to full security in situations like warp and light sync, where state transitions aren't checked.
+Switching the set should be done by issuing an `InitiateChange` event with the parent block hash and the new set, storing the pending set, and then waiting for a call to `finalizeChange` (by the `SYSTEM_ADDRESS`: `2^160 - 2`) before setting the active set to the pending set. This mechanism is used to ensure that the previous validator set "signs off" on the changes before activation, leading to full security in situations like warp and light sync, where state transitions aren't checked.
```
Comment (ChainSafe) [inline EIPS/eip-aura.md]: ...set. Storing the...waiting to call `finalizeChange`...

**Comment by 5chdn** [comment]
LGTMN

**Pull request merged**